circle.yml

```diff
@@ -5,8 +5,6 @@ dependencies:
   cache_directories:
     - /home/ubuntu/nvm/versions/node/v4.5.0/bin
     - /home/ubuntu/nvm/versions/node/v4.5.0/lib/node_modules
-    - node_modules
-    - bower_components
```

Comment:
This will probably slow down our tests a good amount. Couldn't we just reset the cache on circle, somehow?